Contributing.md

```diff
@@ -0,0 +1,96 @@
+# Node.js Community Contributing Guide 1.0
+
+## Code of Conduct
+
+The Code of Conduct explains the *bare minimum* behavior
+expectations the Node.js Foundation requires from all contributors.
+[Please read it before participating.](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
```

Comment:
Maybe point to the one in this repo ? That way if it gets update you don't need to update this doc as well.